content/concepts/content-routing.md

```diff
@@ -3,6 +3,112 @@ title: Content Routing
 weight: 5
 ---
 
-This article is coming soon!
+Peer-to-peer networks are inherently tricky to route because of the following 
+interdisciplinary routing challenges:
 
-Please [refer to this issue](https://github.com/libp2p/docs/issues/23) to track the progress and make suggestions.
+- The lack of universal orchestration that a central server can provide when 
+  querying and retrieving content.
+- Not having a central directory that contains information about reaching every peer 
+  in the network.
+- The presence of high node churn.
+- Data classification is communication-expensive.
+- Creating a resilient, scalable, and optimal routing protocol that is resistant to 
+  Sybil attacks and node churn while being future-proof.
+
+Content requests are made in content addressing through the CID.
+If we want to retrieve a file, we send a general request to the network with the CID.
+
+In libp2p, content routing is used for advertising content-addressed chunks of data.
+There exists a content routing interface with operations that allows 
+for content lookups.
+
+Peer routing schemes may be required to find the peers on the network based on the content 
+routing protocol; DHT-based routing is an example of this, where a DHT is used as a namespace 
+that contains peer information about how to contact different peers on the network. 
+See the peer routing guide for more details.
```

Comment:
I'd put this in a note, as it's not really related to Content Routing but is just clearing up some potential confusion, and also link to the guide.

```suggestion
DHTs are also commonly used for resolving Peer IDs to addresses, which is separate from Content Routing. See the Peer Routing Guide for more details.
```